mlflow/transformers.py

```diff
@@ -1461,12 +1444,24 @@ def _predict(self, data):
             conversation_output = self.pipeline(self._conversation)
             return conversation_output.generated_responses[-1]
         elif isinstance(data, dict):
-            raw_output = self.pipeline(**data)
+            if self.inference_config:
+                raw_output = self.pipeline(**data, **self.inference_config)
```

Comment:
This was the other bug (part 2) where the inference configuration was not being applied to the wrapped `generate()` method of the pipeline. Now it is (confirmed with a note in the test suites)